python/paddle/fluid/tests/unittests/test_reduce_op.py

```diff
@@ -387,6 +393,7 @@ class TestProd6DOp(OpTest):
     def setUp(self):
         self.op_type = "reduce_prod"
         self.python_api = raw_reduce_prod
+        self.prim_op_type = "prim"
         self.init_data_type()
         self.inputs = {
             'X': np.random.random((5, 6, 2, 3, 4, 2)).astype(self.data_type)
```

Comment:
confirm all dtypes  has been tested (FP16, FP32,Fp64)

Reply:
FP16 is not supported, Fp64 has been added

Reply:
<img width="997" alt="591602a47371af7ac4432f53a88fe2b2" src="https://user-images.githubusercontent.com/22424850/227097289-d89ad79b-c37c-4a20-81c5-044cb0877741.png">